src/core/hle/kernel/physical_core.h

```diff
@@ -49,6 +58,18 @@ class PhysicalCore {
     // Shutdown this physical core.
     void Shutdown();
 
+    bool IsInitialized() const {
+        return !!arm_interface;
```

Comment:
```suggestion
        return arm_interface != nullptr;
```

Is more intuitive to read.